ocaml/xapi/db_gc.ml

```diff
@@ -133,9 +114,10 @@ let check_host_liveness ~__context =
           )
         else if live then (
           debug
-            "Assuming host is offline since the heartbeat/metrics haven't been \
+            "Assuming host %s is offline since the heartbeat hasn't been \
              updated for %.2f seconds; setting live to false"
-            (now -. host_time) ;
+            (Ref.string_of host)
+            (Scheduler.span_to_s elapsed) ;
```

Comment:
It's better to use the native printe for time spans:
```
            "Assuming host %s is offline since the heartbeat hasn't been \
             updated for %s; setting live to false"
            (Ref.string_of host)
            (Fmt.to_to_string Mtime.Span.pp elapsed) ;
```
Or include this commit that includes a nice to remember function for printing it:
https://github.com/xapi-project/xen-api/commit/0c5023711c99e734112a32751b80782a19144b88
and then use the function:
```suggestion
            "Assuming host %s is offline since the heartbeat hasn't been \
             updated for %a; setting live to false"
            (Ref.string_of host)
            Pp.mtime_span elapsed ;